BUILDING.md

```diff
@@ -355,16 +357,16 @@ the `lib` or `src` directories.
 The tests attempt to detect support for IPv6 and exclude IPv6 tests if
 appropriate. If your main interface has IPv6 addresses, then your
 loopback interface must also have '::1' enabled. For some default installations
-on Ubuntu that does not seem to be the case. To enable '::1' on the
+on Ubuntu, that does not seem to be the case. To enable '::1' on the
 loopback interface on Ubuntu:
 
-```bash
+```text
```

Comment:
`bash` can introduce syntax highlighting which we don't want for this. IMO `bash` is for bash script files, not command lines.

This comes up over and over again and I think it's a sign that we really shouldn't be having the "you must use a tag on your fenced code" lint check.

Reply:
I think `shell` would be fine here and perhaps in other places where I use `text` above. It can introduce syntax highlighting too, as it's a synonym for `console`. Unlike `console`, at least to me, it doesn't semantically imply that it shows output and a prompt.  But we don't use it anywhere else I don't think.

Reply:
`sh` was also suggested in the past, which would make more sense to me (but would have the same syntax highlighting issue). `shell` is an alias for `console` in highlight.js, I think it's supposed to be equivalent. If no language tag was an option, I agree it would probably be the right choice for this kind of snippet.